package/src/languageServiceManager/kustoLanguageService.ts

```diff
@@ -1695,7 +1695,7 @@ class KustoLanguageService implements LanguageService {
         const paramSymbol: sym.ScalarSymbol = Kusto.Language.Symbols.ScalarTypes.GetSymbol(
             getCslTypeNameFromClrType(param.type)
         );
-        return new sym.ParameterSymbol(param.name, paramSymbol, null);
+        return new sym.ParameterSymbol(param.name, paramSymbol, param.docstring ?? null);
```

Comment:
Tested this change, others are a bit more speculative

Reply:
I'm not clear on the use case. 
Could we add a unit or integration test to clarify it and help prevent regressions?

Reply:
Examples and docstring/description are added to tooltips on hover. I'm using them for dashboard system parameters, and and I'd assume other ones are in the system we get from kusto (otherwise, IDK why the C# api would expose it)

> Could we add a unit or integration test to clarify it and help prevent regressions?

Unlike a lot of other stuff, this isn't really load bearing. If it breaks it won't block a release. I don't think it should be a priority for testing.

Reply:
If a unit test would help I might add that, because it would be easy, but I don't think it would do anything useful.

An integration tests would be pretty expensive to add.

Reply:
I'm sorry if I wasn't clear. 
From my perspective, testing is an important part of our work as engineers. 
It helps ensure that what we build remains reliable and understandable for other developers. 
I would suggest adding an integration test here, but if that's too difficult, a unit test would also be fine.

Reply:
@maxburs can you add some examples/screenshots of there this shows up and how?
That would help in understanding how/what test we should add.

Reply:
<img width="618" alt="image" src="https://github.com/user-attachments/assets/63213c75-95d3-42bd-9030-5dccd6eaae1f">